modules/specialized-guides/pages/salt/salt-ssh.adoc

```diff
@@ -39,7 +47,7 @@ Salt SSH supports both password and key authentication.
 
 Password authentication is used only when bootstrapping.
 During the bootstrap step the key of the server is not authorized on the client and therefore a password must be used for a connection to be made.
-The password is used transiently in a temporary roster file used for bootstrapping.
+The password is used transiently in a temporary roster file used for bootstrapping and the roster file is removed on finishing processing the event.
```

Comment:
IMO this change is not needed since "temporary roster file" implies that it's being removed afterwards.

Reply:
Some temporary files could be left untouched on the system and from the security point of view it's better to mention it explicitly.

Reply:
You're right that "temporary" only means "not forever" and could imply a file is kept for 1 day / week / year etc. If you'd rather be explicit, that's fine for me :+1: